src/App.tsx

```diff
@@ -4,15 +4,14 @@ import { HomePage } from './Pages/HomePage'
 import { RoleSelectOnboard } from './components/RoleSelectionOnboard/RoleSelectionOnBoard'
 import { Route, Switch } from 'react-router'
 import { Paths } from './components/helpers/routeHelper'
+import { LoginOnboard } from './components/LoginOnboard/LoginOnboard'
```

Comment:
Login is a page so please rename LoginOnboard to just LoginPage and put it into Pages folder